### DISCUSSION TITLE
Running shiori as either a daemon or service file

### DISCUSSION THREAD

**Discussion by xjmzx** [go-shiori/shiori#989] [Q&A] [answered]
Hi there. Sorry if this has previously been answered. When runinng `shiori server` it starts in terminal an obviously that terminal window needs to stay alive unless using tmux or something. I just wanted to know if it's possible to start quietly as a daemon, maybe from there it would be possible to automate on reboot with a systemd service file (linux) or launchctl (macOS)

Shiori is a project I've started to love, especially as it keeps improving. I find automation helps to adopt and use projects I love as a daily driver.

**Comment by mjanssens** [comment, marked as answer]
Hi, I saw your question and since I recently started using shiori it was easy to answer.
You can find your systemd unit file documentation in de current docs:
https://github.com/go-shiori/shiori/blob/master/docs/Frequently-Asked-Question.md#how-to-run-shiori-on-start-up-

Hope this helps.

**Comment by xjmzx** [comment]
Ah! Silly me. I'm sure I read the documents. Problem solved. Thaks a lot.
